src/liveEditor/utils/multipleElementAddButton.ts

```diff
@@ -198,28 +204,28 @@ export function handleAddButtonsForMultiple(
 
 export function removeAddInstanceButtons(
```

Comment:
The add instance button is a large module. Let's create a folder in the `liveEditor/` folder and keep all the related code, types, etc together.